EventsExpress/ClientApp/src/components/event/event-form.js

```diff
@@ -84,7 +81,7 @@ class EventForm extends Component {
     }
 
     resetForm = () => {
```

Comment:
This callback is used to clear image. It is resetting whole form. Should be specific to the image, not the form. Consider moving logic of clearing the image into the control itself, where the button is.